react/features/youtube-player/components/native/EnterVideoLinkPrompt.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+
+import { InputDialog } from '../../../base/dialog';
+import { connect } from '../../../base/redux';
+import AbstractEnterVideoLinkPrompt from '../AbstractEnterVideoLinkPrompt';
+
+/**
+ * Implements a component to render a display name prompt.
+ */
+class EnterVideoLinkPrompt extends AbstractEnterVideoLinkPrompt<*> {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        return (
+            <InputDialog
```

Comment:
Can you set this as the placeholder? https://youtu.be/TB7LlM4erx8 you can use the `textInputProps` prop for that.